btree/btree.c

```diff
@@ -2503,12 +2558,13 @@ static int m0_btree_put_makespace_phase(struct m0_btree_op *bop)
 	 */
 
 	/* 1)Move some records from current node to new node */
-	node_move(curr_level->l_node, curr_level->l_alloc, D_LEFT, NR_EVEN,
-		  bop->bo_tx);
-
 	/* Update level for newly allocated */
+	curr_level->l_alloc->n_skip_inv = true;
```

Comment:
Why `n_skip_inv` is needed ?

Reply:
with level >0, the invariant checks if level >0, then no. of records should be greater than 0, with newly allocated internal node, it will fail, and we can not set level of newly allocated node as 0 to make invariant successful, Moreover, when there is a situation where we want to move all the records from one internal node to other , the internal node will become empty and it will fail this check if level>0, rec>0, therefore in such scenario , we required to skip such invariant check
 as level is required for other invariant check , refer line no. 1970, ff_rec_is_valid